HousingSearchApi.Tests/V1/E2ETests/Fixtures/AssetFixture.cs

```diff
@@ -18,22 +18,22 @@ public class AssetFixture : BaseFixture
         public static AddressStub[] Addresses =
         {
             new AddressStub{ FirstLine = "59 Buckland Court St Johns Estate", AssetType = "Dwelling", PostCode = "N1 5EP", UPRN = "10008234650",
-                AssetStatus = "Reserved", NoOfBedSpaces = 2, NoOfCots = 1, HasStairs = true, PrivateBathroom = true, PrivateKitchen = true, StepFree = true, ParentAssetIds = GetGuids() },
+                AssetStatus = "Reserved", NoOfBedSpaces = 2, NoOfCots = 1, HasStairs = true, PrivateBathroom = true, PrivateKitchen = true, StepFree = true, ParentAssetIds = GetGuids(), ContractIsApproved = false},
```

Comment:
`ContractIsApproved` property will be missing from existing records, so it might be worth updating one of the fixtures to not have that property at all. By making that change the result for `ServiceFiltersFalseContractIsApprovedStatusWithoutSearchText` test for example shouldn't be affected at all since the property should be ignored altogther in the query.